test/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package e2e_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestE2E(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "E2E Suite")
+}
```

Comment:
It's not super clear to me whether we'll have a top-level e2e suite vs. a per-provisioner e2e suite, but this is probably fine for now until we get a better feel for the repository.

Reply:
🤔 Yeah that's a good point. I would like to think we'd have a per-provisioner suite now that you mention it. It would be hard to bootstrap a suite into the two existing provisioners and would set a good precedent. Would you me want to move forward on that?